src/components/workspace-utils.js

```diff
@@ -103,7 +103,7 @@ export const SnapshotInfo = ({
   // Lifecycle
   Utils.useOnMount(() => {
     const loadSnapshotInfo = async () => {
-      const snapshotInfo = await Ajax(signal).DataRepo.snapshot(snapshotId).details()
+      const snapshotInfo = await Ajax(signal).DataRepo.snapshot(snapshot).details()
```

Comment:
This looks like a type mismatch to me—providing a value named "snapshot" to a function that names the parameter "snapshotId." It may be worth investigating where this mismatch is introduced and either commenting there to make the intention explicit or fixing the error if that is what has occurred.